static/js/settings_bots.js

```diff
@@ -83,6 +83,50 @@ exports.generate_flaskbotrc_content = function (email, api_key) {
            "\n";
 };
 
+exports.bots_allowed_for_non_admins_values = {
+    Everyone: {
+        code: 1,
+    },
+    Restricted: {
+        code: 2,
+    },
+    'Admins only': {
+        code: 3,
+    },
+};
+
+exports.update_bot_settings_tip = function () {
+    var permission_type = exports.bots_allowed_for_non_admins_values;
+    var current_permission = page_params.realm_bots_allowed_for_non_admins;
+    var tip_text;
+    if (current_permission === permission_type['Admins only'].code) {
+        tip_text = "Only organization administrators can add bots to this organization";
+    } else if (current_permission === permission_type.Restricted.code) {
+        tip_text = "Only orgainzation administrators can add generic bots";
+    } else {
+        tip_text = "Anyone in this organization can add bots";
+    }
+    $(".bot-settings-tip").text(i18n.t(tip_text));
```

Comment:
@timabbott Is this wrong for translation?

Reply:
where is this text appearing?

Reply:
Yes, it's wrong for translation.  You need i18n.t("Anyone in this organization can add bots").  The reason is that the `i18n.t()` syntax has two purposes: First, to basically act as a dictionary lookup to replace untranslated strings with translated strings in production (which this code does), and second to mark the string so that `manage.py makemessages` can extra it from the code so that it gets into the data set of strings to be translated (which this code does not).  So in practice this sort of approach doesn't work.

Reply:
Okay, this cleared how i18n() works, I wasn't aware of the second use case. Thanks.